Source/JavaScriptCore/dfg/DFGSpeculativeJIT.cpp

```diff
@@ -9674,53 +9674,61 @@ void SpeculativeJIT::compileNewArray(Node* node)
     size_t scratchSize = sizeof(EncodedJSValue) * node->numChildren();
     ScratchBuffer* scratchBuffer = vm().scratchBufferForSize(scratchSize);
     EncodedJSValue* buffer = scratchBuffer ? static_cast<EncodedJSValue*>(scratchBuffer->dataBuffer()) : nullptr;
-
-    for (unsigned operandIdx = 0; operandIdx < node->numChildren(); ++operandIdx) {
+    switch (node->indexingType()) {
         // Need to perform the speculations that this node promises to perform. If we're
         // emitting code here and the indexing type is not array storage then there is
         // probably something hilarious going on and we're already failing at all the
         // things, but at least we're going to be sound.
-        Edge use = m_graph.m_varArgChildren[node->firstChild() + operandIdx];
-        switch (node->indexingType()) {
-        case ALL_BLANK_INDEXING_TYPES:
-        case ALL_UNDECIDED_INDEXING_TYPES:
-            CRASH();
-            break;
         case ALL_DOUBLE_INDEXING_TYPES: {
```

Comment:
Let's deindent the switch's case and block because of coding style guide https://webkit.org/code-style-guidelines/